Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "wadm"
 description = "wasmCloud Application Deployment Manager: A tool for running Wasm applications in wasmCloud"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Note: We decided to change this to 0.7.0 if I recall correctly @thomastaylor312

Reply:
Yes that is correct. Feel free to bump here, but I know I have one PR I should open today